mikeio/spatial/FM_geometry.py

```diff
@@ -1258,8 +1258,11 @@ def elements_to_geometry(
             else:
                 geom._type = self._type
                 geom._n_layers = n_layers
-                lowest_sigma = self.n_layers - self.n_sigma_layers + 1
```

Comment:
wouldn't be enough to change this line to (remove the +1):

    lowest_sigma = self.n_layers - self.n_sigma_layers